### PR TITLE
Reduced wait times for secure Nifi Deploy and delete

### PR DESCRIPTION
**Pull request by rguderian**
Signed-off-by: Roger Guderian <roger.guderian@ibm.com>

### CODE REVIEW

File: test-umbrella/tests/tests/run-fvttests.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo " change to the correct deployment directory"
 cd /workspace/$TEST_NAMESPACE/health-patterns/helm-charts/health-patterns
 
 # increase the time to wait for the deploy to be ready (25 minutes)
```
Comment (ibmatc) [inline test-umbrella/tests/tests/run-fvttests.sh:19]: update "25 minutes" to new value?

File: test-umbrella/tests/tests/run-ivttests.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ echo " change to the correct deployment directory"
 cd /workspace/$TEST_NAMESPACE/health-patterns/helm-charts/health-patterns
 
 # increase the time to wait for the deploy to be ready (25 minutes)
```
Comment (ibmatc) [inline test-umbrella/tests/tests/run-ivttests.sh:19]: update "25 minutes" to new value?

**Review by ibmatc — APPROVED** [review]
LGTM

just a couple minor comment suggestions

**Review by ranum — APPROVED** [review]
LGTM

File: test-umbrella/tests/tests/run-ivttests.sh (hunk 2 above)
Comment (rguderian) [reply test-umbrella/tests/tests/run-ivttests.sh]: VoM9hBQU6T

**Pull request merged**